CHANGELOG.md

```diff
@@ -9,16 +9,41 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
-# Changes Since v3.3.0
+# Changes Since v3.4.0
 
-## New features / functionalities
+# v3.4.0 - [2019.08.23]
 
-  - Added support for multiline variables in singularity def-files
+## New features / functionalities
+  
+  - New support for building and running encrypted containers with RSA keys and passphrases
+    - `--pem-path` option added to the `build` and action commands for RSA based encrypted containers
+    - `--passphrase` option added to `build` and action commands for passphrase based encrypted containers
+    - `SINGULARITY_ENCRYPTION_PEM_PATH` and `SINGULARITY_ENCRYPTION_PASSPHRASE` environment variables added to serve same functions as above
+    - `--encyrpt` option added to `build` command to build an encrypted container when environment variables contain a secret
+  - New `--disable-cache` flag prevents caching of downloaded containers
+  - Added support for multi-line variables in singularity def-files
   - Added support for 'indexed' def-file variables (like arrays)
```

Comment:
if this is a new release we should not recycle past release bulletpoints right?

Reply:
Crap.  Did I carry over stuff from an older release?  Let me look.

Reply:
Wait, no, those were under the `Changes since 3.3.0` heading.  Which I took to mean that a responsible developer had actually added their changes to the changelog.  Now the changes since 3.3 are changes in 3.4.  Does that make sense?